addons/html_editor/static/tests/banner.test.js

```diff
@@ -15,10 +15,10 @@ test("should insert a banner with focus inside followed by a paragraph", async (
 
     await press("enter");
     expect(getContent(el)).toBe(
-        `<p>Test</p><div class="o_editor_banner user-select-none o_not_editable lh-1 d-flex align-items-center alert alert-info pb-0 pt-3" role="status" contenteditable="false">
+        `\u200b<div class="o_editor_banner user-select-none o_not_editable lh-1 d-flex align-items-center alert alert-info pb-0 pt-3" role="status" contenteditable="false">
```

Comment:
I don't think there should be a `U200B` in any of those tests.

Reply:
https://github.com/odoo/odoo/blob/master/addons/html_editor/static/src/main/banner_plugin.js#L101C1-L104C10

This code inserts a `\u200B` character if a banner is the first child of an editable.